tf_files/configs/kube-setup-fence.sh

```diff
@@ -62,12 +63,28 @@ if [[ -f "${WORKSPACE}/${vpc_name}_output/creds.json" ]]; then # update secrets
 
   if ! g3kubectl get secrets/fence-json-secret > /dev/null 2>&1; then
     if [[ ! -f "./apis_configs/fence_credentials.json" ]]; then
-      cp "${GEN3_HOME}/tf_files/configs/fence_credentials.json" "./apis_configs/fence_credentials.json" 
+      cp "${GEN3_HOME}/tf_files/configs/fence_credentials.json" "./apis_configs/fence_credentials.json"
     fi
     echo "create fence-json-secret using current creds file apis_configs/fence_credentials.json"
     g3kubectl create secret generic fence-json-secret --from-file=fence_credentials.json=./apis_configs/fence_credentials.json
   fi
 
+  if ! kubectl get secrets/fence-google-app-creds-secret > /dev/null 2>&1; then
```

Comment:
Would you switch this to use `g3kubectl` ?  It's a little wrapper that respects the `KUBECTL_NAMESPACE` environment variable.  It's not really necessary here (since Jenkins doesn't create secrets), but I'd like to stay consistent (I have another branch that fixes the other calls :-p)